packages/main/src/components/Loader/Loader.cy.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { LoaderType } from '../../enums';
+import { Loader } from './index';
+
+describe('Loader', () => {
+  it('indeterminate', () => {
+    cy.mount(<Loader data-testid="loader" />);
+    cy.findByTestId('loader').should('have.css', 'animation-duration', '1.2s');
+
+    cy.mount(<Loader data-testid="loader" type={LoaderType.Indeterminate} />);
+    cy.findByTestId('loader').should('have.css', 'animation-duration', '1.2s');
+  });
+  it('determinate', () => {
+    cy.mount(<Loader type={LoaderType.Determinate} data-testid="loader" />);
+    cy.findByTestId('loader')
+      .should('have.css', 'animation-duration', '0s')
+      .should('have.css', 'background-size', '0px');
+
+    cy.mount(<Loader type={LoaderType.Determinate} data-testid="loader" progress="50%" />);
+
+    cy.findByTestId('loader')
+      .should('have.css', 'animation-duration', '0s')
+      .should('have.css', 'background-size', '50%');
+  });
+  it('with delay', () => {
+    cy.mount(<Loader delay={300} data-testid="loader" />);
+    cy.findByTestId('loader', { timeout: 200 }).should('not.exist');
+    cy.findByTestId('loader').should('be.visible');
+  });
+
+  //todo pass through props test
```

Comment:
Why no pass through props test here?

Reply:
It's not yet available on the main branch, but only [here](https://github.com/SAP/ui5-webcomponents-react/pull/4040/files#diff-19071bcf809aa76dfe0f43ab79ea831f5ebe247ecb2879eea51774c6c32f87b5).

The comment is there so we don't forget to add it to all tests afters it's available.